src/Task.vue

```diff
@@ -9,6 +9,19 @@
         {{ task.title }}
       </h5>
       <p>{{ task.body }}</p>
+      <p>
+        date : <input type="date" v-model.sync="task.date"></input>
+        location : <select v-model.sync="task.location">

```

Comment:
select tag need label tag.

matteriarize is restrict any html code .
[check for this page](http://materializecss.com/forms.html#select)